spec/unit/puppet/type/firewalld_ipset_spec.rb

```diff
@@ -79,6 +69,29 @@
       provider.expects(:execute_firewall_cmd).with(['--ipset=whitelist', '--remove-entry=10.8.8.8'], nil)
       provider.entries=(['192.168.2.2', '10.72.1.100'])
     end
+  end
+  context 'change in ipset members' do
+    let(:resource) do
+      Puppet::Type.type(:firewalld_ipset).new(
+        name: 'white',
+        type: 'hash:net',
+        entries: ['8.8.8.8/32', '9.9.9.9']
+      )
+    end
 
+    it 'removes /32 in set members' do
+      expect(resource[:entries]).to eq ['8.8.8.8', '9.9.9.9']
```

Comment:
For future reference I'd avoid using public IPs. [RFC5735](https://tools.ietf.org/html/rfc5735) describes subnets exactly for this use case:
> 192.0.2.0/24 - This block is assigned as "TEST-NET-1" for use in documentation and example code.

There's also TEST-NET-2 (198.51.100.0/24) and TEST-NET-3 (203.0.113.0/24).